sunbird-rc-esignet-integration-impl/README.md

```diff
@@ -1,44 +1,52 @@
-# mock-esignet-integration-impl
+# sunbird-rc-esignet-integration-impl
 
 ## About
 
-Implementation for all the interfaces defined in esignet-integration-api. This libaray is built as a wrapper for [mock-identity-system](mock-identity-system) service.
+Implementation for all the interfaces defined in esignet-integration-api. This libaray is built as a wrapper for [sunbird-registry-system](sunbird-registry-url) service.
 
 This library should be added as a runtime dependency to [esignet-service](https://github.com/mosip/esignet) for development purpose only.
 Note: This is not production use implementation.
 
 ## Configurations required to added in esignet-default.properties
 
 ````
-mosip.esignet.integration.scan-base-package=io.mosip.esignet.mock.integration
-mosip.esignet.integration.authenticator=MockAuthenticationService
+mosip.esignet.integration.scan-base-package=io.mosip.esignet.sunbirdrc.integration
+mosip.esignet.integration.authenticator=SunbirdRCAuthenticationService
+mosip.esignet.integration.vci-plugin=SunbirdRCVCIssuancePlugin
 mosip.esignet.integration.key-binder=MockKeyBindingWrapperService
 mosip.esignet.integration.audit-plugin=LoggerAuditService
 mosip.esignet.integration.captcha-validator=GoogleRecaptchaValidatorService
 
-mosip.esignet.send-otp.captcha-required=true
+
 mosip.esignet.captcha-validator.url=https://www.google.com/recaptcha/api/siteverify
 mosip.esignet.captcha-validator.secret=${esignet.captcha.secret.key}
 mosip.esignet.captcha-validator.site-key=${esignet.captcha.site.key}
 
-mosip.esignet.mock.authenticator.get-identity-url=https://${mosip.api.public.host}/v1/mock-identity-system/identity
-mosip.esignet.mock.authenticator.kyc-auth-url=https://${mosip.api.public.host}/v1/mock-identity-system/kyc-auth
-mosip.esignet.mock.authenticator.kyc-exchange-url=https://${mosip.api.public.host}/v1/mock-identity-system/kyc-exchange
-mosip.esignet.mock.authenticator.send-otp=https://${mosip.api.public.host}/v1/mock-identity-system/send-otp
-mosip.esignet.mock.supported.bind-auth-factor-types={'WLA'}
-mosip.esignet.mock.authenticator.ida.otp-channels=email,phone
+##--------------------sunbird registry auntentication releted demo configuration-------------------------##
+
+mosip.esignet.authenticator.sunbird-rc.auth-factor.kba.individual-id-field='policyNumber'
+mosip.esignet.authenticator.sunbird-rc.auth-factor.kba.field-details={{"id":"policyNumber", "type":"text", "format":""},{"id":"name", "type":"text", "format":""},{"id":"dob", "type":"date", "format":"dd/mm/yyyy"}}
+mosip.esignet.authenticator.sunbird-rc.auth-factor.kba.registry-search-url=http://10.3.148.107/registry/api/v1/Insurance/search
```

Comment:
Below config is missing mosip.esignet.authenticator.sunbird-rc.kba.entity-id-field

Reply:
added